CMakeLists.txt

```
@@ -69,10 +64,18 @@ function(trusted_firmware_build)
 
 
   set(VENEERS_FILE ${TFM_BINARY_DIR}/secure_fw/s_veneers.o)
-  set(${TFM_OUT_VENEERS_FILE} ${VENEERS_FILE} PARENT_SCOPE)
-  set(${TFM_OUT_INCLUDE_PATH} ${TFM_BINARY_DIR}/install/export/tfm/include PARENT_SCOPE)
-
   set(PSA_API_NS_PATH ${TFM_BINARY_DIR}/interface/libpsa_api_ns.a)
+  set(TFM_EXPORTED_INCLUDES ${TFM_BINARY_DIR}/install/export/tfm/include)
+  set(SID_H_FILE_PATH ${TFM_EXPORTED_INCLUDES}/psa_manifest/sid.h)
```

Can be changed to:
```suggestion
${TFM_BINARY_DIR}/generated/interface/include/psa_manifest/sid.h
```
if we want to disable the install step.

---

Will do